services/bettersocial/post/commentChild.js

```diff
@@ -221,14 +222,14 @@ const BetterSocialCreateCommentChildV3Anonymous = async (req) => {
     if (body?.message?.length > 80) {
       await countProcess(reaction_id, {comment_count: +1}, {comment_count: 1});
     }
-
-    await sendMultiDeviceReplyCommentNotification(
-      signedUserIdFeed,
-      commentAuthor,
-      message,
-      reaction_id,
-      postTitle
-    );
+    if (!(await UsersFunction.checkIsMe(User, signedUserIdFeed, userId)))
+      await sendMultiDeviceReplyCommentNotification(
+        signedUserIdFeed,
+        commentAuthor,
+        message,
+        reaction_id,
+        postTitle
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous hunk, consider adding error handling here as well to ensure smooth execution of the code even when there are errors in `UsersFunction.checkIsMe` or `sendMultiDeviceReplyCommentNotification`.
```diff
-    if (!(await UsersFunction.checkIsMe(User, signedUserIdFeed, userId)))
-      await sendMultiDeviceReplyCommentNotification(
-        signedUserIdFeed,
-        commentAuthor,
-        message,
-        reaction_id,
-        postTitle
-      );
+    try {
+      if (!(await UsersFunction.checkIsMe(User, signedUserIdFeed, userId))) {
+        await sendMultiDeviceReplyCommentNotification(
+          signedUserIdFeed,
+          commentAuthor,
+          message,
+          reaction_id,
+          postTitle
+        );
+      }
+    } catch (error) {
+      console.error('Error while checking user or sending notification:', error);
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->